config/gen.go

```diff
@@ -0,0 +1,45 @@
+//go:build ignore
+// +build ignore
+
+// This is used with `go generate` to copy schema.json into schema.go.
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+)
+
+const (
+	schemaFile = "schema.json"
+	outputFile = "schema.go"
+)
+
+func main() {
+	schema, err := ioutil.ReadFile(schemaFile)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// "To convey to humans and machine tools that code is generated,
+	// generated source should have a line that matches the following
+	// regular expression (in Go syntax):
+	//
+	//     ^// Code generated .* DO NOT EDIT\.$
+	//
+	// This line must appear before the first non-comment, non-blank
+	// text in the file.":
+	content := "// Code generated by gen.go. DO NOT EDIT.\n\n" +
+		"package " + os.Getenv("GOPACKAGE") + "\n\n" +
+		"var schema = `" + strings.TrimSpace(string(schema)) + "`\n"
```

Comment:
could we use `go:embed` to get `schema.json` available as a variable instead?

Reply:
> could we use go:embed 

Oh, TIL. Yeah, that's much simpler.